frontend/express/public/localization/pre-login/pre-login.properties

```diff
@@ -15,6 +15,10 @@ login.locked = Your account was locked by admin
 login.button = Login
 login.token-expired = Your session was expired. Please login again
 
+#report
+report.unsubscribe-successful = You have successfully unsubscribed from this email report.
+report.unsubscribe-failed = Email report unsubscribe failed.
```

Comment:
Can you recheck this text with Mariano?

Reply:
And would really want to see them inside plugin instead of core.
You can load external localization files in prelogin, like this: https://github.com/Countly/countly-server/blob/master/plugins/enterpriseinfo/frontend/public/templates/login.html#L82